.github/workflows/setup-kicad-library-utils.yml

```diff
@@ -0,0 +1,38 @@
+name: setup-kicad-library-utils
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  test:
```

Comment:
Perhaps, let us create 2 tests:
1. Default case (same as what we have now)
2. One that uses `v8` as the version input for testing backwards compatibility